posthog/migrations/0259_pak_v2.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 3.2.15 on 2022-09-20 17:07
+
+from django.contrib.auth.hashers import get_hasher
+from django.db import migrations, models
+
+PERSONAL_API_KEY_SALT = "posthog_personal_api_key"
+
+
+def hash_key_value(value: str) -> str:
+    return get_hasher().encode(value, PERSONAL_API_KEY_SALT)
+
+
+def hash_all_keys(apps, schema_editor):
+    PersonalAPIKey = apps.get_model("posthog", "PersonalAPIKey")
+    updated_instances = PersonalAPIKey.objects.all()
```

Comment:
Yeah, it will take a couple of minutes for the migration to go through… but that shouldn't be _too_ big of a problem, since the long part is all in Django and not Postgres.

Reply:
I'd still be tempted to filter so `updated_instances` is only objects where `value` is present. If we timeout and retry then we don't want to reprocess all. Very defensive but safer :)

Reply:
Hmm, as far as I understand, if we time out no key will be updated, because migrations are wrapped in transactions (with some nuance to it, but a single step definitely is wrapped). Does that not hold?

Reply:
Ah, doh, yep, because we're not batching the update 👍 

Nit picking complete :)